globus_action_provider_tools/flask/helpers.py

```diff
@@ -0,0 +1,48 @@
+from enum import Enum
+from typing import Iterable, List, Set
+
+from flask import request
+
+
+def parse_query_args(
```

Comment:
can we make the `flask.request` object a parameter here so that we don't have so much reliance on the global-ish state? Perhaps allows for some weird future where we have a request object that's been created by some other means than the incoming call on a flask route.